elements/lisk-transactions/src/14_unlock_transaction.ts

```diff
@@ -95,12 +95,12 @@ export class UnlockTransaction extends BaseTransaction {
 		const errors = [];
 
 		for (const unlock of this.asset.unlockObjects) {
-			if (unlock.amount <= BigInt(0)) {
```

Comment:
original logic is correct amount cannot be less than or equal to `0`

Reply:
yes, but validator checks for amount to be not negative.. so its not required?

Reply:
Even if validator checks for negative it would be nice for this part of the business logic to be maintained in the transaction itself so as to it to be more clear on its working.